tenable/io/v3/was/plugins/api.py

```diff
@@ -0,0 +1,46 @@
+'''
+Plugins
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Web Application Scanning v3 plugins <was-v2-plugins>` API.
+
+Methods available on ``tio.v3.was.plugins``:
+
+.. rst-class:: hide-signature
+.. autoclass:: PluginsAPI
+    :members:
+'''
+from typing import Dict
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class PluginsAPI(ExploreBaseEndpoint):
+    _path = 'api/v3/was/plugins'
+    _conv_json = True
+
+    def plugin_details(self, id: int) -> Dict:
+        '''
+        Returns details for the specified Tenable.io
+        Web Application Scanning plugin.
+
+        :devportal:`was plugins: plugin details <was-v2-plugins-details>`
+
+        Args:
+            id (int):
+                The ID of a Tenable.io Web Application Scanning plugin.
+
+        Returns:
+            :obj:`dict`:
+                The resource record of the was plugin.
+
+        Examples:
+            >>> folder = tio.v3.was.plugins.plugin_details(1)
```

Comment:
I think this should be
plugin = tio.v3.was.plugins.plugin_details(1)

Reply:
updated